### DISCUSSION TITLE
npx in shebang comment

### DISCUSSION THREAD

**Discussion by Gpx** [google/zx#286] [Help needed]
I started adding this shebang comment at the beginning of my scripts:

```sh
#!/usr/bin/env npx --yes zx
```

It ensures that anyone with `npx` installed can run them without having to install `zx`.

I don't see any drawbacks, but I didn't see it documented anywhere, and I was wondering if anyone knows of any issue with this approach.

**Comment by antonmedv** [comment]
Yes, it’s possible. But unfortunately not all systems support arguments in shbang.

**Reply by Gpx** [reply]
Ah, I see. I was not aware of this. I'll test it out with the systems I use then and remember this if I need to build something for general purposes.

Thanks!
